src/components/top-bar/styles.ts

```diff
@@ -46,7 +46,7 @@ export const ElTopBar = styled.div`
     }
   }
 
-  ${isTabletOrBelow} {
+  ${isBelowDesktop} {
```

Comment:
same as above

> 

I initially overlooked this because there are no < or > indicators in the design. However, I now believe this is correct since the only viewport with an indicator is mobile (<768px). This implies that tablet starting at 768px should mean 768px++, and similarly, desktop should be 1024px++, and so on.